clang/include/clang/AST/ExternalASTSource.h

```diff
@@ -145,12 +146,18 @@ class ExternalASTSource : public RefCountedBase<ExternalASTSource> {
   /// Find all declarations with the given name in the given context,
   /// and add them to the context by calling SetExternalVisibleDeclsForName
   /// or SetNoExternalVisibleDeclsForName.
-  /// \return \c true if any declarations might have been found, \c false if
-  /// we definitely have no declarations with tbis name.
+  /// \param DC since the same namespace can appear in multiple module units.
```

Comment:
Isn't this just the `DeclContext` that's used for lookup? Do we need to mention namespaces and multiple modules here? Maybe we can leave that for the `NamedModule` description

Reply:
Yes, done.